.github/workflows/e2e-tests.yml

```
@@ -29,8 +29,9 @@ jobs:
             ~/.cargo/registry/index/
             ~/.cargo/registry/cache/
             ~/.cargo/git/db/
-            target/
+            ./target/
           key: ${{ runner.os }}-cargo-${{ hashFiles('Cargo.lock') }}
```

I think we also would have to do something like:

```suggestion
          key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
```

But I'm not entirely sure.

---

I'm not sure that in this case it will look at the toplevel 'Cargo.lock' file. It is mostly for a project with many lockfiles but we have a single one.

---

yeah, you're right 👍🏻